ci-templates/gitlab/image-builder.yml

```diff
@@ -59,6 +59,18 @@ variables:
   JUPYTER_VERSION:
     value: "python-3.11"
     description: "Python version for the jupyter notebook."
+  T4C_SERVER_REGISTRY:
+    value: ""
+    description: "Registry where you can find the t4c server image"
+  T4C_SERVER_TAG:
+    value: "$CAPELLA_VERSION-main"
+    description: "Tag that is used for the t4c server image"
+  T4C_SERVER_TEST_DATA_REPO:
+    value: ""
+    description: "Link to the t4c test data repo needed to run the backup tests"
+  LOCAL_GIT_BASE_IMAGE:
+    value: "debian:bullseye"
+    description: "Specifies the base images used to build the local git server needed to run the tests"
```

Comment:
I'd not define these images during runtime and instead just set it as global variable in the `.gitlab-ci.yml` of the individual repositories where the template is included.

Reply:
Done